cygwin/Platform.c

```diff
@@ -0,0 +1,337 @@
+/*
+htop - cygwin/Platform.c
+(C) 2014 Hisham H. Muhammad
+(C) 2015 Michael McConville
+(C) 2015 David C. Hunt
+(C) 2017,2018 Guy M. Broome
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "cygwin/Platform.h"
+#include "cygwin/CygwinMachine.h"
+
+#include <math.h>
+#include <stdio.h>
+
+#include "CPUMeter.h"
+#include "ClockMeter.h"
+#include "DateMeter.h"
+#include "DateTimeMeter.h"
+#include "HostnameMeter.h"
+#include "LoadAverageMeter.h"
+#include "Macros.h"
+#include "MemoryMeter.h"
+#include "MemorySwapMeter.h"
+#include "Meter.h"
+#include "XUtils.h"
+#include "Settings.h"
+#include "SwapMeter.h"
+#include "SysArchMeter.h"
+#include "TasksMeter.h"
+#include "UptimeMeter.h"
+
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
```

Comment:
Please be consistent about how you handle the `WIN32_LEAN_AND_MEAN` flag.

Reply:
> Please be consistent about how you handle the `WIN32_LEAN_AND_MEAN` flag.

got it

Reply:
> Please be consistent about how you handle the `WIN32_LEAN_AND_MEAN` flag.

please check it again, does it looks better now?